Yoast/javascript
# Test the performance of a Polish dictionary stemmer

## Summary


This PR can be summarized in the following changelog entry:

* Adds a button to the content analysis app that tests the performance of the Polish stemmer (compared to the English and Spanish stemmers)

## Relevant technical choices:

* The papers used in the performance test are added to the full text tests papers; even though they differ in purpose from the other full text tests
* All the papers use the same Russian language text and metadata, the only thing that changes between the papers is the locale (this allows for optimal comparison)

## Test instructions

This PR can be tested by following these steps:

* Run the content analysis app and open the console
* There should be a button that says "Compare performance Polish/Spanish/English" under 'Controls'. Click on it.
* In the console, there should appear performance statistics for Polish, Spanish, and English. See image below for example (the actual statistics may differ):
<img width="498" alt="Screenshot 2020-10-12 at 17 10 11" src="https://user-images.githubusercontent.com/32479012/95762701-5e3f9480-0cae-11eb-988b-fc23574cac6c.png">

**Note**: After clicking the performance analysis button, there may also appear errors in the console. There will be another issue about fixing them, so as long as everything else works these errors can be ignored.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-234?atlOrigin=eyJpIjoiZWZjMTQwMDZhMmI2NDlmOWJiMzgwMGM1NjVjZTQ1OGIiLCJwIjoiaiJ9
